src/continuous_agg.h

```diff
@@ -53,6 +53,9 @@ ts_continuous_agg_hypertable_status(int32 hypertable_id);
 extern void ts_continuous_agg_drop_chunks_by_chunk_id(int32 raw_hypertable_id, Chunk **chunks,
 													  Size num_chunks);
 extern TSDLLEXPORT List *ts_continuous_aggs_find_by_raw_table_id(int32 raw_hypertable_id);
+extern TSDLLEXPORT List *ts_continuous_aggs_find_by_raw_table_id_with_skip(int32 raw_hypertable_id,
```

Comment:
I don't see the implementation of this function anywhere.

Reply:
left over from earlier version. removed.

Reply:
removed. leftover from previous version